src/index.css

```diff
@@ -1,13 +1,13 @@
 body {
   margin: 0;
+  padding-top: 20px;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
-}
+* {
+  box-sizing: border-box;
+}
```

Comment:
nit picking: end of file :P

Reply:
fixed in https://github.com/chatbotgang/image-map-editor/pull/30/commits/a49fa14208f1fe814698ec8b1c23decbe605ebab